admin/server/projects.go

```diff
@@ -1828,30 +1828,40 @@ func (s *Server) TriggerRedeploy(ctx context.Context, req *adminv1.TriggerRedepl
 	return &adminv1.TriggerRedeployResponse{}, nil
 }
 
-func (s *Server) projToDTO(p *database.Project, orgName string) *adminv1.Project {
+func (s *Server) projToDTO(ctx context.Context, p *database.Project, orgName string) *adminv1.Project {
+	var createdByUserEmail string
+	if p.CreatedByUserID != nil {
+		u, err := s.admin.DB.FindUser(ctx, *p.CreatedByUserID)
+		// ignoring err will be empty in case error in getting user
+		if err == nil {
+			createdByUserEmail = u.Email
+		}
+	}
```

Comment:
It should only return the creator email for admin users, not safe to show to public/reader users